Update Getting Stated Docs for Windows Server 2019 (#2832)
openenclave/openenclave: docs/GettingStartedDocs/Contributors/WindowsSGX1GettingStarted.md

Note: To check if your system has support for SGX1, please look [here](../SGXSupportLevel.md).

A version of Windows OS with native support for SGX features:
- For server: Windows Server 2016 or 2019
- For client: Windows 10 64-bit version 1709 or newer
- To check your Windows version, run `winver` from the command line

## Install Git and Clone the Open Enclave SDK repo

- Download and install Git for Windows from [here](https://git-scm.com/download/win).
- Clone the Open Enclave SDK to a folder of your choice. In these instructions
  we're assuming `C:/Users/test`.

```powershell

> cd C:/Users/test/

Reviewer: Just for my understanding, why do we specify a longer path here now? Were folks encountering issues with just using the project rooted in C:\?

Reviewer 2: In WindowsSGX1FLCGettingStarted.md, the path was change during 0.8 to c:/users/test. I believe the reason was that the  clone path was  c:\openenclave and then the package also got installed to c:\openenclave which created a lot of confusion. Changing it here to c:\users\test is good since it keeps both docs consistent.

./scripts/install-windows-prereqs.ps1 -InstallPath C:/oe_prereqs -LaunchConfiguration SGX1 -DCAPClientType None
```

> On Windows 10, the Intel PSW is delivered via Windows Update, and on Windows 

Reviewer 3: Ditto.

Run the command `powershell.exe` to open a PowerShell prompt within the native
tools environment.

From here, use CMake and Ninja to build/install Open Enclave.

To build debug enclaves:

```powershell
cd C:/Users/test/openenclave
mkdir build/x64-Debug
cd build/x64-Debug

> cmake -G Ninja -DHAS_QUOTE_PROVIDER=OFF -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCMAKE_INSTALL_PREFIX=install ../..

Reviewer 2: cmake -G Ninja -DHAS_QUOTE_PROVIDER=OFF -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCMAKE_INSTALL_PREFIX=install ../..
```suggestion
cmake -G Ninja -DHAS_QUOTE_PROVIDER=OFF -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCMAKE_INSTALL_PREFIX=C:/openenclave ../..
```

> cmake -G Ninja -DHAS_QUOTE_PROVIDER=OFF -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCMAKE_INSTALL_PREFIX=install ../..

Reviewer 2: ```suggestion
cmake -G Ninja -DHAS_QUOTE_PROVIDER=OFF -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCMAKE_INSTALL_PREFIX=C:/openenclave ../..
```

ninja
```

Similarly, to build release enclaves, specify the flag
`-DCMAKE_BUILD_TYPE=Release`:

```powershell
cd C:/Users/test/openenclave
mkdir build/x64-Release
cd build/x64-Release

> cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DHAS_QUOTE_PROVIDER=OFF -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCMAKE_INSTALL_PREFIX=install ../..

Reviewer 2: ```suggestion
cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DHAS_QUOTE_PROVIDER=OFF -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCMAKE_INSTALL_PREFIX=c:/openenclave ../..
```

```
Refer to the [LVI Mitigation](AdvancedBuildInfo.md#lvi-mitigation) documentation for further information.

Now, using the `ninja install` command will install the SDK in

> `C:/openenclave`. To choose a different location, change

Reviewer: Is `C:\openenclave` the expected default install path?

Given that the build happens under `C:/Users/test/openenclave/build/x64-Release` and ` -DCMAKE_INSTALL_PREFIX=install` is specified, wouldn't the expected install path be `C:/Users/test/openenclave/build/x64-Release/install`?

Reviewer 2: Thanks for pointing this out. If my suggestion at line #86 is committed, the install path would be c:\openenclave

A clean pass of the above unit tests is an indication that your Open Enclave setup was successful.

For more information refer to the [Advanced Test Info](AdvancedTestInfo.md) document.

## Build and run samples

> To build and run the samples independently of SDK building/installation, please look [here](/samples/README_Windows.md).

Reviewer: Dave has previously suggested not linking non-descriptive words like "here" or "this". Consider instead:
```suggestion
To build and run the samples without building and then installing the OE SDK, please refer to the [README for Windows samples](/samples/README_Windows.md).
```
